.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
+  - 5.4 #Drop it??
```

Comment:
Yes, please, let's drop PHP 5.4 as we're anyway going to drop support for it soon.

Reply:
droped